api/src/main/java/org/openmrs/module/insuranceclaims/InsuranceClaimsActivator.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.insuranceclaims;
+
+import org.openmrs.module.BaseModuleActivator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class contains the logic that is run every time this module is either started or shutdown
+ */
+public class InsuranceClaimsActivator extends BaseModuleActivator {
+	
+	private static final Logger log = LoggerFactory.getLogger(InsuranceClaimsActivator.class);
+	
+	/**
+	 * @see #started()
+	 */
+	public void started() {
```

Comment:
is this method overrides BaseModuleActivation method ? if yes add an annotation

Reply:
Added, thanks.